### PR TITLE
[BPK-1697] Add gears-manual and gears-automatic icons

### PR DESCRIPTION
**Pull request by shaundon**
I went for `gears-manual` and `gears-automatic`, plus aliased the old ones to `gears-manual-circle` and `gears-automatic-circle`. 

So now we have:

| Icon        | Names           |
| ------------- |:-------------:|
| screen shot 2018-07-12 at 09 00 20      | transmission-automatic, gears-automatic-circle |
| screen shot 2018-07-12 at 09 00 24 | transmission-manual, gears-manual-circle      |
| screen shot 2018-07-12 at 09 01 57 | gears-manual      |
| screen shot 2018-07-12 at 09 02 00 | gears-automatic |

 The inconsistency is a bit annoying but it's better than a breaking change, right?

### CODE REVIEW

**Comment by k0nserv** [comment]
So with this we'll have 

+ `transmission-automatic` (Circle)
+ `transmission-manual` (Circle)
+ `gears-automatic`
+ `gears-manual`

?

**Comment by shaundon** [comment]
@k0nserv that's right. Not the most optimal but it seems the lesser of two evils for consumers. I'm very open to other names too, of course - gears was just my first thought!

**Comment by georgegillams** [comment]
Can we add an alias so that `transmission-automatic-cicle` === `transmission-automatic`

**Comment by georgegillams** [comment]
And also so that `gears-automatic-circle` === `transmission-automatic-circle`

**Comment by georgegillams** [comment]
I'm just thinking that if someone has `gears-automatic` in their code-base, they'd expect to replace it with the circle version by adding `-circle` to the end

**Comment by shaundon** [comment]
Yeah that sounds sensible to me @georgegillams.

So then we'd have:

| Icon        | Names           |
| ------------- |:-------------:|
| screen shot 2018-07-12 at 09 00 20      | transmission-automatic, transmission-automatic-circle, gears-automatic-circle |
| screen shot 2018-07-12 at 09 00 24 | transmission-manual, transmission-manual-circle, gears-manual-circle      |
| screen shot 2018-07-12 at 09 01 57 | gears-manual      |
| screen shot 2018-07-12 at 09 02 00 | gears-automatic |

**Comment by matthewdavidson** [comment]
The public names should be:

| Icon        | Names           |
| ------------- |:-------------:|
| screen shot 2018-07-12 at 09 00 20      | gears-automatic-circle |
| screen shot 2018-07-12 at 09 00 24 | gears-manual-circle      |
| screen shot 2018-07-12 at 09 01 57 | gears-manual      |
| screen shot 2018-07-12 at 09 02 00 | gears-automatic |

With the existing names `transmission-automatic`, `transmission-manual` only existing for backwards compat.

No point in adding the `-circle` variants for the existing names.

File: unreleased.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Unreleased
 
+**Added:**
+- bpk-svgs
```
Comment (matthewdavidson) [inline unreleased.md]: Way more packages are affected...

- `bpk-component-icon`
- `react-native-bpk-component-icon`
- `react-native-bpk-component-button`
- (anything that exposes icons directly)


Check the changelog for previous addition to icons to see an example

Comment (shaundon) [reply unreleased.md]: Thanks - now fixed!

**Pull request merged**